containers/LocationMarker.js

```diff
@@ -5,19 +5,48 @@ import { MapView } from 'expo';
 import PinCallout from '../components/PinCallout';
 
 class LocationMarker extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.handleDragStart = this.handleDragStart.bind(this);
```

Comment:
I think you can make the methods below lambdas and then not need to bind them like this:

`handleDragStart = (event) => { ...`

Reply:
I've been trying to move away from using lambdas so much. I have read articles that they tend to be a lot slower than bound functions. And since these functions will get called quite often (sometimes rapidly) I thought it might be good to have them be bound instead of lambdas. What do you think?

Reply:
(When I was writting typescript we would use the `@autobind` decoraters which was super nice)

Reply:
Oh neat! Please send me your articles on the topic! Looking at babel it seemed fine ... https://babeljs.io/docs/en/babel-plugin-proposal-class-properties